ros/foxglove_msgs/package.xml

```diff
@@ -19,7 +19,7 @@
   <build_depend condition="$ROS_VERSION == 2">rosidl_default_generators</build_depend>
   <build_depend>ros_environment</build_depend>
 
-  <depend>visualization_msgs</depend>
+  <depend>builtin_interfaces</depend>
```

Comment:
Hm - this seems like it should be conditional on ROS 2. If we don't plan to ever publish for ROS 1 again, then we should remove all the conditional stuff.

Reply:
Good catch, updated.

I doubt we will publish any more ROS 1 updates but we can remove all that in another PR.

Reply:
We should still be considerate of folks building ROS 1 things from source.